packages/blade/src/components/Form/FormLabel.tsx

```diff
@@ -53,6 +59,7 @@ const FormLabel = ({
   children,
   id,
   htmlFor,
+  size = 'medium',
```

Comment:
ensure the default was medium before also, otherwise could break in cases where we used this component without specifying the size prop

Reply:
Yeah default was medium only